inspire/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -593,6 +612,45 @@
                 }
             }
         },
+        "succeeding_entry": {
+            "type": "object",
+            "properties": {
+                "relationship_code": {
```

Comment:
It is a bit difficult to know what this field is meant for. A description would be useful.